lib/show_n_tell_web/controllers/auth_controller.ex

```diff
@@ -11,6 +11,18 @@ defmodule ShowNTellWeb.AuthController do
     |> render_result(conn)
   end
 
+  def show(conn, _params) do
```

Comment:
Nice - showing authorization!

Reply:
Yes, this controller action will allow us to retrieve a user based on an access token